wpeverest/user-registration
# UR-872 Enhance - Integration menu for email marketing.

### All Submissions:






### Changes proposed in this Pull Request:



Previously, The Integration menu was for Captcha Settings, Now we have moved captcha into Captcha Menu and now Integration is for email marketing addons like MailChimp, MailerLite, ActiveCampaign, Klaviyo.

### Design 
https://www.figma.com/proto/pGx46GyMMoRzTeRme4A6fg/UR-Settings?page-id=2463%3A24598&node-id=2463-27916&viewport=5421%2C-9611%2C0.22&scaling=min-zoom

### Dependent PR's
MailChimp - https://github.com/wpeverest/user-registration-mailchimp/pull/26
MailerLite - https://github.com/wpeverest/user-registration-mailerlite/pull/7

### How to test the changes in this Pull Request:

1. Verify Whether Backward Compatibility for captcha is working or not as I have modified captcha option names.
2. Only if (MailChimp, MailerLite, ActiveCampaign) one of these addon is activated then only integration tab will be shown.
3. Open Integration menu and check backward compatibility for previously connected accounts as some of option names are changed.
4. Check new accounts are adding and it listing in the form builder -> form settings-> mailchimp properly.
5. Check email marketing are working properly.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Integration menu for email marketing.
